backend/src/accounts.ts

```diff
@@ -75,10 +75,10 @@ export const getAccountInfo = async (accountId: string) => {
   return {
     accountId: accountInfo.account_id,
     createdByTransactionHash:
-      accountInfo.created_by_transaction_hash || undefined,
+      accountInfo.created_by_transaction_hash || "Genesis",
     createdAtBlockTimestamp: accountInfo.created_at_block_timestamp
       ? parseInt(accountInfo.created_at_block_timestamp)
-      : undefined,
+      : 0,
```

Comment:
On a similar note as above, `0` is now have a special meaning. Similarly to "Genesis", I usually prefer not to mix legal values with some special values.